example-deploy-nodejs-app.md

```diff
@@ -0,0 +1,63 @@
+# Jenkinsfile to deploy nodejs app
+
+Here is a sample Jenkinsfile to deploy nodejs app
+
+```
+#!/usr/bin/groovy
+@Library('github.com/stakater/fabric8-pipeline-library@deploy-nodejs')
```

Comment:
Shouldn't this be changed back to master?

Reply:
good catch will change

Reply:
ideally it should point to a "released" tag as then it will be secure as otherwise we might keep changing it
for now i will point it to master